### PR TITLE
Add In-Page HorizontalTabView with first iteration of new design

### PR DESCRIPTION
**Pull request by duke7553**
Borrowing from the same idea which allows the vertical tabs flyout to exist, we now add the horizontal multitasking control directly onto ModernShellPage.

In the future, this change will require us to have the horizontal tab view be disabled automatically when Panes are in use.

This PR updates the design of the tabs based on user feedback. We didn't like how the titlebar and sidebar clashed so the old acrylic titlebar is gone in favor of a different drag region and content now extended directly upwards. This change, pictured below enhances how pronounced tabs are in light/dark mode while retaining our undocked design. The selected tab now has a better looking drop shadow which increases ever so slightly on hover to invite the user to manipulate the tab further.

image
image

### CODE REVIEW

File: Files/UserControls/MultitaskingControl/HorizontalMultitaskingControl.xaml [hunk 1]
```diff
@@ -113,7 +114,8 @@
                                             <ContentPresenter
                                                 x:Name="RightContentPresenter"
                                                 Grid.Column="3"
-                                                Margin="0,0,250,0"
+                                                Margin="0,0,0,0"
```
Comment (yaichenbaum) [inline Files/UserControls/MultitaskingControl/HorizontalMultitaskingControl.xaml:117]:
```suggestion
```

File: Files/UserControls/MultitaskingControl/HorizontalMultitaskingControl.xaml [hunk 2]
```diff
@@ -124,7 +126,8 @@
                                             x:Name="ShadowCaster"
                                             Grid.Row="0"
                                             Height="10"
-                                            Margin="0,0,0,-10"
+                                            Visibility="Collapsed"
+                                            Margin="0,0,0,0"
```
Comment (yaichenbaum) [inline Files/UserControls/MultitaskingControl/HorizontalMultitaskingControl.xaml:130]:
```suggestion
```

**Review by yaichenbaum — APPROVED** [review]

**Review by yaichenbaum — APPROVED** [review]

**Comment by yaichenbaum** [comment]
@duke7553 Great work! LGTM

**Comment by gave92** [comment]
This looks great! One question though:
here the TabStrip was moved from MainPage to ModernShellPage, wasn't it better to move the Sidebar from ModernShellPage to MainPage instead? So there where no issues with the dual pane view.

**Pull request merged**